Formula/htmltest.rb

```diff
@@ -0,0 +1,29 @@
+class Htmltest < Formula
+  desc "HTML validator written in Go"
+  homepage "https://github.com/wjdp/htmltest"
+  url "https://github.com/wjdp/htmltest/archive/v0.14.0.tar.gz"
+  sha256 "add922cf1dd957afba2927d401184c1d2331983a6d8ed96dd10f5001930cebf8"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
```

Comment:
Sorry, I just noticed that they use [these build flags](https://github.com/wjdp/htmltest/blob/master/build.sh):
```
go build -ldflags "-X main.date=`date -u +%Y-%m-%dT%H:%M:%SZ` -X main.version=`git describe --tags`" -o bin/htmltest -x main.go
```
Let's try to replicate some of this as follows:
```suggestion
    ldflags = %W[
      -X main.date=#{Utils.safe_popen_read("date", "-u", "+%Y-%m-%dT%H:%M:%SZ")}
      -X main.version=#{version}
    ]
    system "go", "build", *std_go_args, "-ldflags", ldflags.join(" ")
```

Reply:
Thanks, sorry for not spotting that myself! I've put that into the formula now. This has fixed the `-v` actually outputting the version correctly now 😅